ledger/metrics/src/main/scala/com/daml/metrics/PruningMetrics.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.metrics.api.MetricName
+import com.daml.metrics.api.MetricHandle.Factory
+
+class PruningMetrics(prefix: MetricName, factory: Factory) {
+  // Using a meter, which can keep track of how many time the operation was executed, even if the
+  // operation is fully executed between 2 metric fetches by the monitoring.
+  // With a (boolean) gauge, there is a large risk that some operation invocation would be missed.
+  val pruneStarted = factory.meter(prefix :+ "prune" :+ "started")
```

Comment:
Nit: To be grammatically correct, I'd suggest renaming these metrics to `pruneRequestStarted` or `pruningStarted` (and `pruneRequestCompleted` or `pruningCompleted`, respectively).

Reply:
I renamed the variables to `pruneCommandStarted`.
I kept the metric name to `daml_services_pruning_prune_started_total`. To me, it makes sense, it is the `prune` method of the `pruning` service.